### PR TITLE
Util fixes

### PR DESCRIPTION
**Pull request by ozancaglayan**
## Description

- Formatting fixes
- Removes the spurious `isinstance()` check (#1020)
- Fix format string error for `PredictionException`

## Type of change

- [X] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: ts/utils/util.py
Comment (msaroufim) [inline ts/utils/util.py:107]: since we're here do you mind changing this to an `f` string

File: ts/utils/util.py
Comment (msaroufim) [inline ts/utils/util.py:48]: we can improve this to show the mapping file path

File: ts/utils/util.py
Comment (msaroufim) [inline ts/utils/util.py:71]: how about List[str] instead of [str]

**Review by msaroufim — CHANGES_REQUESTED** [review]
Thank you! Since we're here might as well improve some of the logging messages otherwise easy to merge

File: ts/utils/util.py
Comment (ozancaglayan) [reply ts/utils/util.py:107]: yeah sure, I was wondering whether you are being conservative for earlier python versions, otherwise I find f-strings much more concise. Does this mean that, I should replace `%s` style substitutions with f-strings if I hit them?

File: ts/utils/util.py
Comment (ozancaglayan) [reply ts/utils/util.py]: done

File: ts/utils/util.py
Comment (ozancaglayan) [reply ts/utils/util.py]: done

File: ts/utils/util.py
Comment (msaroufim) [reply ts/utils/util.py]: Yeah for security reasons we support only Python 3.8+. I've been wanting to do all these upgrades but it's easier to manage if we do it per new file that someone happened to touch

**Review by msaroufim — APPROVED** [review]

**Comment by ozancaglayan** [comment]
should we merge this or do you want to wait for a 2nd approval?

**Review by rohithkrn — APPROVED** [review]

**Pull request merged**
